en/html/README.md

```diff
@@ -167,10 +167,10 @@ Make sure you're in the `djangogirls` directory and let's tell `git` to include
 
 {% filename %}command-line{% endfilename %}
 ```
-$ git add --all .
+$ git add .
 ```
 
-> __Note__ `--all` means that `git` will also recognize if you've deleted files (by default, it only recognizes new/modified files). Also remember (from chapter 3) that `.` means the current directory.
+> __Note__ By default, `git add` will recognize if you've deleted files (by default, it only recognizes new/modified files). Also remember (from chapter 3) that `.` means the current directory.
```

Comment:
Do you recall which version of git changed this?

Reply:
https://github.com/git/git/blob/e31aba42fb12bdeb0f850829e008e1e3f43af500/Documentation/RelNotes/2.0.0.txt#L106

Reply:
Check line 106

Reply:
I'm getting old. Somehow I thought git 2.0 was much more recent. Given it was released in 2014 I think we can safely assume everyone has it by now.

Reply:
(Resolved conversation, as it seems unlikely that Git versions from before 2014 are in use by programming newcomers.)